src/applications/app.helpers.js

```diff
@@ -52,10 +52,6 @@ export function shouldntBeActive(app) {
   }
 }
 
-export function notBootstrapped(app) {
```

Comment:
was this a public API function or just used internally?

Reply:
just internally